tests/metrics/test_github.py

```diff
@@ -0,0 +1,225 @@
+import pytest
+from app.source.github import Github
+from app.daos.dao_git_metric import dao_upsert_git_metric
+from github3.exceptions import NotFoundError
+from datetime import datetime
+
+
+def mock_github_client(mocker, prs_dict=None, commit_dates_arr=None):
+    if not commit_dates_arr:
+        commit_dates_arr = [
+            {
+                'date': "2018-12-10T12:12:12",
+                'files': [{'additions': 20, 'deletions': 10}]
+            },
+            {
+                'date': "2018-12-11T12:12:12",
+                'files': [{'additions': 20, 'deletions': 10}]
+            },
+            {
+                'date': "2018-12-12T12:12:12",
+                'files': [{'additions': 20, 'deletions': 10}]
+            },
+            {
+                'date': "2018-12-13T12:12:12",
+                'files': [{'additions': 20, 'deletions': 10}]
+            },
+            {
+                'date': "2018-12-14T12:12:12",
+                'files': [{'additions': 20, 'deletions': 10}]
+            }
+        ]
+
+    class PullReq:
+        comments_count = 0
+        review_comments_count = 4
```

Comment:
Would be good if we can also move the `comments_count` and `review_comments_count` into the `pr_dict`